README_js.md

```diff
@@ -17,157 +17,77 @@ require('crypto').randomBytes = function() {
+  - Chrome, Safari, Firefox, Edge, IE 11 browsers
+  - Webpack, Rollup packagers
+- **Secure** - Cryptographically-strong random values
+- **Small** - Zero-dependency, small footprint, plays nice with "tree shaking" packagers
+- **CLI** - Includes the [`uuid` command line](#command-line) utility
 
-- Support for version 1, 3, 4 and 5 UUIDs
-- Cross-platform: CommonJS build for Node.js and [ECMAScript Modules](#ecmascript-modules) for the
-  browser.
-- Uses cryptographically-strong random number APIs
-- Zero-dependency, small footprint
-
-⚠️⚠️⚠️ This is the README of the upcoming major version of this library. You can still [access the README
-of the current stable version](https://github.com/uuidjs/uuid/blob/v3.4.0/README.md). ⚠️⚠️⚠️
-
-## Upgrading from v3.x of this Library
-
-The latest major version of this library is v7.x and the previous major version was v3.x. We
-decided to jump v4.x, v5.x and v6.x in order to avoid any confusion with [version
-4](#version-4-random) and [version 5](#version-5-namespace) UUIDs and the [version 6 UUID
-proposal](http://gh.peabody.io/uuidv6/).
-
-### Deep Requires now Deprecated
-
-In v3.x of this library we were promoting the use of deep requires to reduce bundlesize for browser
-builds:
-
-```javascript
-const uuidv4 = require('uuid/v4'); // <== NOW DEPRECATED!
-uuidv4();
-```
-
-As of v7.x this library has been converted to ECMAScript Modules and deep requires are now
-deprecated and may be removed in a future major version of this library.
-
-Since all modern bundlers like rollup or Webpack support tree-shaking for ECMAScript Modules out of
-the box we now encourage you to use modern `import` syntax instead, see [ECMAScript Modules /
-ESM](#ecmascript-modules--esm):
-
-```javascript
-import { v4 as uuidv4 } from 'uuid';
-uuidv4();
-```
-
-For use as CommonJS module with Node.js you can use:
-
-```javascript
-const { v4: uuidv4 } = require('uuid');
-uuidv4();
-```
-
-### Default Export Removed
-
-v3.x of this library was exporting the Version 4 UUID method as a default export:
-
-```javascript
-const uuid = require('uuid'); // <== REMOVED!
-```
-
-This usage pattern was already discouraged in v3.x and has been removed in v7.x.
+**Upgrading from uuid\@3?** Your code is probably okay, but check out [Upgrading
+From uuid\@3](#upgrading-from-uuid3) for details.
 
 ## Quickstart - Node.js/CommonJS
 
 ```shell
 npm install uuid
 ```
 
-Then generate a random UUID (v4 algorithm), which is almost always what you want ...
+Once installed, decide which type of UUID you need. RFC4122 provides for four
+versions, all of which are supported here. In order of popularity, they are:
+
+- Version 4 (random) - Created from cryptographically-strong random values
+- Version 1 (timestamp) - Created from the system clock (plus random values)
+- Version 5 (namespace, SHA-1) - Created from user-supplied name and namespace strings
+- Version 3 (namespace, MD5) - Like version 5, above, but with a poorer hash algorithm
```

Comment:
Link to the API docs below?

Reply:
I don't think this is needed.  This is in the context of the "Quick Start", which provides examples immediately below this.  Pointing people at the detailed (more confusing) API docs doesn't feel helpful here.